format/index.md

```diff
@@ -261,9 +261,9 @@ relationship isn't currently associated with any target resources.
 
 Resource linkage **MUST** be represented as one of the following:
 
-* `null` for empty to-one relationships.
-* a [resource identifier object] for non-empty to-one relationships.
-* an empty array (`[]`) for empty to-many relationships.
+* `null`: for empty to-one relationships.
+* `[]`: for empty to-many relationships.
```

Comment:
Same here - probably ok to stick with the old wording about "an empty array (`[]`)"